packages/core/content-type-builder/admin/src/components/ListRow/DisplayedType.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Typography } from '@strapi/design-system/Typography';
+import getTrad from '../../utils/getTrad';
+
+const DisplayedType = ({ readableType, customField, repeatable }) => {
+  const { formatMessage } = useIntl();
+
+  if (customField) {
+    return (
+      <Typography>
+        {formatMessage({
+          id: getTrad('attribute.customField'),
+          defaultMessage: 'Custom field',
+        })}
+      </Typography>
+    );
+  }
+
+  return (
+    <Typography>
+      {formatMessage({
+        id: getTrad(`attribute.${readableType}`),
+        defaultMessage: readableType,
```

Comment:
One small change this introduces is that the defaultMessage is readableType instead of type. But I feel like it makes more sense, I don't understand why the default message would be more precise than the actual translation

Reply:
The defaultMessage is the fallback when a trad id is not defined in a specific locale, here's the [doc](https://formatjs.io/docs/intl#message-descriptor)